Description: Fix conntrack parsing for kernels using lowercase protocol names
Author: Matthijs Kooijman <matthijs@stdin.nl>
Forwarded: yes

--- a/main.cc
+++ b/main.cc
@@ -169,7 +169,7 @@ static int Resolve(void)
             in_addr masqaddr = { byteswap(masqhost) };
             msyslog(ll_debug, " (masq parse) proto = %s, masqhost = %s, l = %u, r = %u, masqport = %u",
                     proto, inet_ntoa(masqaddr), l, r, masqport);
-            if(strcmp(proto, "TCP"))continue;
+            if(strcasecmp(proto, "TCP"))continue;
             if(r == remoteport && masqport == localport)
                 return Forward(masqhost, r, l);
         }
